### DISCUSSION TITLE
KeePassXC-2.7.5-x86_64.AppImage won't open a browser window

### DISCUSSION THREAD

**Discussion by dlpoole** [keepassxreboot/keepassxc#9559] [Q&A] [answered]
KeePassXC-2.7.5-x86_64.AppImage ignores the OpenURL option in an entry's context menu. It neither opens a new browser window nor a new tab in an open one. However, KeePassXC-2.5.4-x86_64.AppImage does so perfectly as I am accustomed to. Both are downloaded side-by-side on Linux Mint 21.1 Cinnamon. The browser is Firefox 114.0.1 (64-bit)

Not a particular problem since I can run the old version, and prefer the look of it, but I'd rather have it work on the new one.

**Comment by droidmonkey** [comment, marked as answer]
#8721
